h2o-core/src/main/java/water/webserver/H2OHttpViewImpl.java

```diff
@@ -2,6 +2,7 @@
 
 import org.apache.commons.io.IOUtils;
 import water.ExtensionManager;
+import water.H2O;
```

Comment:
forgotten import?

Reply:
Yes, thanks. Resolved in https://github.com/h2oai/h2o-3/pull/4554/commits/2e4abce3a2616c5b64bb68ca47d5762bbc7ad94f.